MantleTests/MTLTestModel.m

```diff
@@ -24,7 +24,7 @@ @implementation MTLTestModel
 
 #pragma mark Properties
 
-- (BOOL)validateName:(NSString **)name error:(NSError **)error {
+- (BOOL)validateName:(NSString **)name error:(NSError * __autoreleasing *)error {
```

Comment:
Sorry for that, due to the reason mentioned before, NSString** also need __autoreleasing in this case. I'll create another pull request to fix it. Thanks for pointing this out.